api/api/src/main/scala/uk/ac/wellcome/platform/api/elasticsearch/ElasticsearchQuery.scala

```diff
@@ -292,3 +293,18 @@ final case class BoolBoostedQuery(q: String) extends ElasticsearchPartialQuery {
     must(TitleQuery(q).elasticQuery).boost(1000)
   )
 }
+
+final case class ImageSimilarityQuery(q: String, index: Index)
+    extends ElasticsearchQuery {
+  private final val lshFields = List("inferredData.lshEncodedFeatures")
+  private lazy val documentRef = DocumentRef(index, q)
+
+  lazy val elasticQuery =
+    moreLikeThisQuery(lshFields)
+      .likeDocs(List(documentRef))
+      .copy(
+        minTermFreq = Some(1),
+        maxQueryTerms = Some(1000),
+        minShouldMatch = Some("1")
```

Comment:
Potentially some of these could be default arguments rather than defined inline, I dunno :man_shrugging:

Reply:
Yeah, again I was copying the convention which is that the only arguments are required ones - I think this approach makes sense as it means the queries can be used without having to understand the actual ES query specification that's going on